content/p2p_sync_intro.tex

```diff
@@ -0,0 +1,62 @@
+The following section discusses \openshmem \ac{API}s that provides a mechanism
+for synchronization between two \ac{PE}s based on the value of a symmetric data
+object.
+The point-to-point synchronization routines can be used to portably ensure
+that memory access operations observe remote updates in the order enforced by
+the initiator PE using the \FUNC{shmem\_fence} and \FUNC{shmem\_quiet}
+routines.
+
+Where appropriate compiler support is available, \openshmem provides
+type-generic point-to-point synchronization interfaces via \Celev{} generic
+selection. Such type-generic routines are supported for the
+``point-to-point synchronization types'' identified in
+Table~\ref{p2psynctypes}.
+
+\begin{table}[h]
+  \begin{center}
+    \begin{tabular}{|l|l|}
+      \hline
+      \TYPE              & \TYPENAME  \\ \hline
+      float              & float      \\ \hline
```

Comment:
I don't think we know how to support it. For this reason we should avoid introducing this